docs/custom-json-deserializers.md

```diff
@@ -0,0 +1,29 @@
+# Customizing JSON-B deserializers
+
+If your application needs finer-grained control over JSON deserialization than you can achieve via
+formatting annotations like `@JsonbDateFormat`, you may plug in your own custom instances of the `Jsonb` class
+for each input type that your GraphQL application exposes.
+
+To do this, you'll need to create an implementation of `io.smallrye.graphql.spi.EventingService`
+and implement its `overrideJsonbConfig` method. 
+An example that plugs in a custom date format for a particular class that is used as input:
+
+```
+public class CustomJsonbService implements EventingService {
+
+    @Override
```

Comment:
Should we make this a default method in the interface ?

Reply:
I think this is generally something that the user should be forced to (at least) think about, so I'd probably keep it as it is..